cli/azd/internal/cmd/show/show.go

```diff
@@ -539,6 +539,8 @@ func showTypeFromLanguage(language project.ServiceLanguageKind) contracts.ShowTy
 		return contracts.ShowTypeNode
 	case project.ServiceLanguageJava:
 		return contracts.ShowTypeJava
+	case project.ServiceLanguageCustom:
+		return contracts.ShowTypeCustom
```

Comment:
Can we (cheaply) wrap this in a feature flag?  I think we might want to revisit this in a couple of months when we're further a long with our extension story.